### ISSUE TITLE
Production?

### ISSUE THREAD

**Issue by WiFEED** [IsaacAlves7/docker-templates#2]
Hello can i use this for production ? 
My goal is to set up lamp and lemp servers to host a lot of sites on them

**Comment by IsaacAlves7** [comment]
Yeah, sure! It's just modify some lines of this dockerfile to run for production. I'll make some branches about each Stack explaning about it! Have a good day :)

**Comment by WiFEED** [comment]
> Yeah, sure! It's just edit some lines of this dockerfile to run for production. I'll make some branches in this repository to explain about each Web Stack (LAMP, LEMP, MERN, etc).
> 
> Thank you! And have a good day :)
> 
> Also, the link below can help you about it:
> 
> * https://youtu.be/8vXoMqWgbQQ

Awesomeeeeeeeeee ! 
I don't really know dockerfile, how I could do to change PHP versions for all containers ? Thank you so mutch, have a good day !

**Comment by IsaacAlves7** [comment]
- https://hub.docker.com/_/php?tab=tags

See the tags and choose the best version/environment for your project.

**Comment by WiFEED** [comment]
> * https://hub.docker.com/_/php?tab=tags
> 
> See the tags and choose the best version/environment for your project.

Ok thank you ! :D I have another problem .. 

" ERROR: Service 'php' failed to build: The command '/bin/sh -c apt update && apt install -y     chmod ugo+x /usr/local/bin/install-php-extensions && sync &&     install-php-extensions pdo pdo_mysql gd zip exif' returned a non-zero code: 100 " When i try to build the docker-compose.yml
